docs/gs.md

```diff
@@ -1,4 +1,4 @@
-<h1 align="center"><b>Getting Started...</b></h1> 
+<h1 align="center"><b>Getting Started...</b></h1>
```

Comment:
In the last PR also,  I pointed out please use markdown specific syntax for Headings